src/pybind/mgr/dashboard/controllers/cluster.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from ..security import Scope
+from ..services.cluster import ClusterModel
+from . import ApiController, ControllerDoc, EndpointDoc, RESTController, allow_empty_body
+
+
+@ApiController('/cluster', Scope.CONFIG_OPT)
+@ControllerDoc("Get Cluster Details", "Cluster")
+class Cluster(RESTController):
+    @RESTController.MethodMap(version='0.1')
+    @EndpointDoc("Get the cluster status")
+    def list(self):
+        return ClusterModel.from_db().dict()
+
+    @RESTController.MethodMap(version='0.1')
+    @allow_empty_body
```

Comment:
This explains everything: we're passing the cluster status as a query param. We should change that and use the body to pass the status data.

Reply:
Changed to singleton_set